terraform/aks/config/production.tfvars.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
can add a probe_path now, so
"probe_path": "/healthcheck"

Reply:
@RMcVelia The docs mention `extermal_url` https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/onboard-service.md#enable-monitoring should I include both `probe_path` and `external_url`?

Reply:
It should default to the app_urls/probe_path for statuscake, so you would only set external_url if you want something different or extra